packages/contracts/src/dollar/core/UbiquityDollarToken.sol

```diff
@@ -51,6 +50,14 @@ contract UbiquityDollarToken is ERC20Ubiquity {
         address account,
         uint256 amount
     ) public override onlyDollarBurner whenNotPaused {
+        uint256 currentAllowance = allowance(account, _msgSender());
```

Comment:
Why was the `UbiquityDollarToken` contract modified? And how is this code change connected with the "security monitoring" feature?